### DISCUSSION TITLE
VanJS JavaScript and TypeScript Vite Templates Added To Awesome Vite

### DISCUSSION THREAD

**Discussion by yahia-berashish** [vanjs-org/van#171] [Show and tell]
I've made a PR in Awesome Vite a while ago for adding VanJS JavaScript and TypeScript templates, it got merged just now, this is the link for the PR: 
https://github.com/vitejs/awesome-vite/pull/799

You can view and use the templates from Awesome Vite’s README here: https://github.com/vitejs/awesome-vite
Or directly from the repositories: 
- JavaScript: https://github.com/yahia-berashish/vite-vanjs-js
- TypeScript: https://github.com/yahia-berashish/vite-vanjs-ts
(Feel free to give them a star 😄). 

The templates use the `vanjs-core` and `vanjs-ext` (VanX) packages (I added VanX because I thought a Vite project would be usually "large-scale" enough to need it). 
A PR for VanJSX & TSX templates will hopefully be coming soon 👋

**Comment by Tao-VanJS** [comment]
Thank you so much @yahia-berashish! I added your template to VanJS's [community examples](https://vanjs.org/demo#community-examples).
